applications/nrf5340_audio/src/utils/error_handler.c

```diff
@@ -16,9 +16,14 @@ LOG_MODULE_REGISTER(error_handler, CONFIG_ERROR_HANDLER_LOG_LEVEL);
 
 #if (defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP) && (CONFIG_DEBUG))
 /* nRF5340 Audio DK center RGB LED */
-static const struct gpio_dt_spec center_led_r = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_red), gpios);
-static const struct gpio_dt_spec center_led_g = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_green), gpios);
-static const struct gpio_dt_spec center_led_b = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_blue), gpios);
+static const struct gpio_dt_spec error_led_0 = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_red), gpios);
```

Comment:
I do not think we need this change. This will mean that there will be a single green LED indicating an error. This is counterintuitive to me. If there is an error, the APP core alive LED will stop blinking, and we don't have to make any changes to this module.